### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
Found misspellings.

### CODE REVIEW

File: assets/lib/locales/locales.php [hunk 1]
```diff
@@ -112,7 +112,7 @@ public function numbers_for_index( $index, $how_many = 3, $test_up_to = 1000 ) {
 	public function index_for_number( $number ) {
 		if ( ! isset( $this->_index_for_number ) ) {
 			$gettext = new Gettext_Translations;
-			$expression = $gettext->parenthesize_plural_exression( $this->plural_expression );
+			$expression = $gettext->parenthesize_plural_expression( $this->plural_expression );
```
Comment (szepeviktor) [inline assets/lib/locales/locales.php:115]: @pedro-mendonca How about this one?

Comment (szepeviktor) [reply assets/lib/locales/locales.php:115]: > This method has been deprecated. Use [Plural_Forms()](https://developer.wordpress.org/reference/classes/plural_forms) instead.

Comment (szepeviktor) [reply assets/lib/locales/locales.php:115]: But only for v6.5

Comment (pedro-mendonca) [reply assets/lib/locales/locales.php]: The `locales.php` file is a clone from GlotPress. This should be reported upstream.

**Comment by pedro-mendonca** [comment]
Thanks for the fix 👍

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**